soc/xtensa/esp32_net/linker.ld

```diff
@@ -0,0 +1,438 @@
+/*
+ * Copyright (c) 2022 Espressif Systems (Shanghai) Co., Ltd.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Linker command/script file
```

Comment:
There are lots of changes in current `esp32` linker. I suggest updating this as well.

Reply:
Most of them does nott apply here, I shall remind the memory available for esp32 APP_CPU in a AMP configs is available in IRAM+DRAM only, again this can be reviewed in a new PR when adding support for flash memory